app/views/admin/locations/_location.html.erb

```diff
@@ -9,4 +9,10 @@
     <%= location.longitude %>
   </p>
 
+  <hr>
+
+  <div id="map1"></div>
```

Comment:
@lime666 for the future: could you please use "Friendly Names" for ids, classes, and variables. In this case, for example, it could be `id="location-map"` instead `map1`.